src/apps/routes/schemas/app.clj

```diff
@@ -415,6 +421,50 @@
   (merge AppListing
          {:apps (describe [AdminAppListingDetail] "A listing of App details")}))
 
+(def AppListingValidSortFields
+  (-> (map ->required-key (keys AppListingDetail))
```

Comment:
It might be helpful to create an alias of `->required-key` indicating that the schema key is being converted to a keyword. Something like `schema-key->keyword`.